### DISCUSSION TITLE
How to find same filenames (with different extensions) and delete all other files ?

### DISCUSSION THREAD

**Discussion by gidhap** [WinMerge/winmerge#1074] [General]
I want to clean up some directories with lots of pictures where (in the same directory) I have pairs with same name but different extensions (both to be kept), plus a lot of other files which I do not need any longer.

Example:
eagle.dng       keep
eagle.RW2      keep
test1.jpg         delete
bear.jpeg        delete
moose.dng     delete
caribou.RW2  delete

I am totally new to WinMerge and tried to play with filter but I don't get it yet. Any help appreciated

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the ability to find the same file names.

However, WinMerge can sort by file name, so if you disable Tree Mode and sort by file name, it will be easier to find the same file name.

![image](https://user-images.githubusercontent.com/98126/144941579-712befc4-3e01-4ab7-9fc4-2f6d9892f801.png)
